README.md

```diff
@@ -2570,7 +2571,7 @@ The stack transform supports diverging stacks: negative values are stacked below
 After all values have been stacked from zero, an optional **offset** can be applied to translate or scale the stacks. The following **offset** methods are supported:
 
 - null (default) - a zero baseline
-- *expand* (or *normalize*) - rescale each stack to fill [0, 1]
+- *normalize* (or *expand*) - rescale each stack to fill [0, 1]
```

Comment:
Funny, in #289 I was thinking of *normalize* as an alias for *expand*, not that we were deprecating *expand*. But I agree we should favor one to reduce confusion. In other cases where we’ve deprecated options, we’ve removed the deprecated options from the README entirely. I think we should probably do that here, too—better to not spend the user’s attention on documenting deprecated options.